packages/bridge/src/bridge.ts

```diff
@@ -0,0 +1,77 @@
+import type {
+    WebBridge,
+    CondoBridge,
+    AnyRequestMethodName,
+    RequestParams,
+    RequestId,
+    CondoBridgeSubscriptionListener,
+} from './types/bridge'
+import { promisifySend } from './promisify'
+import pkg from '../package.json'
+
+export const IS_CLIENT_SIDE = typeof window !== 'undefined'
+// TODO(DOMA-5084): add mobile clients env checks
+export const IS_BROWSER_CLIENT = IS_CLIENT_SIDE
+
+const WEB_SPECIFIC_METHODS = [
+    'CondoWebAppResizeWindow',
+]
+
+const ALL_METHODS = [
+    ...(IS_BROWSER_CLIENT ? WEB_SPECIFIC_METHODS : []),
+]
+
+const webBridge: WebBridge | undefined = IS_BROWSER_CLIENT
+    ? parent
+    : undefined
+
+export function createCondoBridge (): CondoBridge {
```

Comment:
Can we extend CondoBridge from EventEmitter?

Reply:
Emitters are useful, when you're emit and subscribe to events from different part of code...In our cases this parts are on separate clients...And we cannot share emitter instance across that parts...So it's partlial emitter :)